Description:
 Upstream author of IE PNG Alpha Fix agreed that every version of his script
 can be distributed under the terms of the LGPL 2.1 or later version.

Index: python-docutils-0.6/docutils/writers/s5_html/themes/default/iepngfix.htc
===================================================================
--- python-docutils-0.6.orig/docutils/writers/s5_html/themes/default/iepngfix.htc	2010-01-07 12:33:26.000000000 +0100
+++ python-docutils-0.6/docutils/writers/s5_html/themes/default/iepngfix.htc	2010-01-07 12:33:49.000000000 +0100
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/default/blank.gif';
Index: python-docutils-0.6/test/functional/expected/ui/default/iepngfix.htc
===================================================================
--- python-docutils-0.6.orig/test/functional/expected/ui/default/iepngfix.htc	2010-01-07 12:33:26.000000000 +0100
+++ python-docutils-0.6/test/functional/expected/ui/default/iepngfix.htc	2010-01-07 12:33:49.000000000 +0100
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/default/blank.gif';
Index: python-docutils-0.6/test/functional/expected/ui/small-black/iepngfix.htc
===================================================================
--- python-docutils-0.6.orig/test/functional/expected/ui/small-black/iepngfix.htc	2010-01-07 12:33:26.000000000 +0100
+++ python-docutils-0.6/test/functional/expected/ui/small-black/iepngfix.htc	2010-01-07 12:33:49.000000000 +0100
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/small-black/blank.gif';
